torchtune/modules/transformer.py

```diff
@@ -576,6 +576,7 @@ def forward(
         encoder_mask: Optional[torch.Tensor] = None,
         input_pos: Optional[torch.Tensor] = None,
         input_embeds: Optional[torch.Tensor] = None,
+        output_mask: Optional[torch.Tensor] = None,
```

Comment:
Comment from @felipemello1 is to not do this and have this done in the loss instead. @pbontrager @ebsmothers @joecummings